sources/packages/forms/src/form-definitions/educationprogram.json

```diff
@@ -1,5871 +1,6243 @@
+                        "strictDateValidation": false,
+                        "multiple": false,
+                        "unique": false,
+                        "onlyAvailableItems": false
+                    },
+                    "key": "hasJointDesignatedInstitution",
+                    "conditional": {
+                        "show": true,
+                        "when": "hasJointInstitution",
+                        "eq": "yes"
+                    },
+                    "type": "radio",
+                    "input": true,
+                    "placeholder": "",
+                    "prefix": "",
+                    "customClass": "",
+                    "suffix": "",
+                    "multiple": false,
+                    "defaultValue": null,
+                    "protected": false,
+                    "unique": false,
+                    "persistent": true,
+                    "hidden": false,
+                    "clearOnHide": true,
+                    "refreshOn": "",
+                    "redrawOn": "",
+                    "modalEdit": false,
+                    "dataGridLabel": false,
+                    "labelPosition": "top",
+                    "description": "",
+                    "errorLabel": "",
+                    "tooltip": "",
+                    "hideLabel": false,
+                    "tabindex": "",
+                    "disabled": false,
+                    "autofocus": false,
+                    "dbIndex": false,
+                    "customDefaultValue": "",
+                    "calculateValue": "",
+                    "calculateServer": false,
+                    "widget": null,
+                    "attributes": {},
+                    "validateOn": "change",
+                    "overlay": {
+                        "style": "",
+                        "left": "",
+                        "top": "",
+                        "width": "",
+                        "height": ""
+                    },
+                    "allowCalculateOverride": false,
+                    "encrypted": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "properties": {},
+                    "allowMultipleMasks": false,
+                    "addons": [],
+                    "inputType": "radio",
+                    "fieldSet": false,
+                    "id": "ecjqzzq"
+                },
+                {
+                    "label": "Join Program Error",
+                    "className": "alert alert-warning fa fa-exclamation-circle",
+                    "attrs": [
+                        {
+                            "attr": "",
+                            "value": ""
+                        }
+                    ],
+                    "content": "<strong> This program is ineligible for StudentAid BC funding.</strong><br/>\nAll partner institutions must be designated by StudentAid BC..",
```

Comment:
double full stop at the end

Reply:
I did not make any adjustment on this content.